.appveyor.yml

```diff
@@ -1,21 +1,27 @@
+image: Visual Studio 2017
+
+clone_depth: 10
+
 environment:
-  nodejs_version: "8"
```

Comment:
:ok_hand: Node 8 is already the [default on all images including VS2017](https://www.appveyor.com/docs/windows-images-software/#node-js) (I'm just documenting my understanding so other reviewers don't have to go through the same process)